### PR TITLE
chore(ci): improve action skipping for internal repository

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: .github/workflows/core_crypto_gpu_benchmark.yml [hunk 1]
```diff
@@ -20,7 +20,8 @@ jobs:
   setup-instance:
     name: Setup instance (cuda-core-crypto-benchmarks)
     runs-on: ubuntu-latest
-    if: ${{ (github.event_name == 'push' && github.repository == 'zama-ai/tfhe-rs') || github.event_name == 'workflow_dispatch' }}
+    if: github.event_name != 'schedule' ||
+      (github.event_name == 'schedule' && github.repository == 'zama-ai/tfhe-rs')
```
Comment (IceTDrinker) [inline .github/workflows/core_crypto_gpu_benchmark.yml:24]: that would trigger bench on push to internal ?

Comment (soonum) [reply .github/workflows/core_crypto_gpu_benchmark.yml:24]: There no `push` event on this one. It's a copy/paste error.

Comment (IceTDrinker) [reply .github/workflows/core_crypto_gpu_benchmark.yml:24]: indeed

File: .github/workflows/integer_gpu_benchmark.yml [hunk 2]
```diff
@@ -178,7 +179,9 @@ jobs:
     name: Slack Notification
     needs: [ setup-instance, cuda-integer-benchmarks ]
     runs-on: ${{ needs.setup-instance.outputs.runner-name }}
-    if: ${{ !success() && !cancelled() }}
+    if: ${{ !success() &&
+      !cancelled() &&
+      (github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'zama-ai/tfhe-rs')) }}
```
Comment (IceTDrinker) [inline .github/workflows/integer_gpu_benchmark.yml:184]: same thing likelly not required to update

File: .github/workflows/integer_gpu_full_benchmark.yml [hunk 3]
```diff
@@ -171,7 +172,9 @@ jobs:
     name: Slack Notification
     needs: [ setup-instance, cuda-integer-full-benchmarks ]
     runs-on: ${{ needs.setup-instance.outputs.runner-name }}
-    if: ${{ !success() && !cancelled() }}
+    if: ${{ !success() &&
```
Comment (IceTDrinker) [inline .github/workflows/integer_gpu_full_benchmark.yml:175]: same

File: .github/workflows/integer_multi_bit_gpu_benchmark.yml [hunk 4]
```diff
@@ -180,7 +181,9 @@ jobs:
     name: Slack Notification
     needs: [ setup-instance, cuda-integer-multi-bit-benchmarks ]
     runs-on: ${{ needs.setup-instance.outputs.runner-name }}
-    if: ${{ !success() && !cancelled() }}
+    if: ${{ !success() &&
```
Comment (IceTDrinker) [inline .github/workflows/integer_multi_bit_gpu_benchmark.yml:184]: same

**Review by IceTDrinker — APPROVED** [review]
thanks !

**Pull request merged**